src/core/test_helpers/kbn_server.ts

```diff
@@ -208,7 +200,7 @@ export function createTestServers({
     defaultsDeep({}, settings.es ?? {}, {
       log,
       license,
-      password: license === 'trial' ? DEFAULT_SUPERUSER_PASS : undefined,
+      password: license === 'trial' ? 'changeme' : undefined,
```

Comment:
I think you can remove this line entirely, as `createTestEsCluster` will default to `'changeme'` if `password` is undefined anyway: https://github.com/elastic/kibana/blob/9da9b8d4b9fec6ef17038ca753885ecd6ed19016/packages/kbn-test/src/es/test_es_cluster.ts#L45-L48